package.json

```diff
@@ -44,7 +44,8 @@
         "onLanguage:json",
         "onLanguage:jsonc",
         "onCommand:azurerm-vscode-tools.uninstallDotnet",
-        "onCommand:azurerm-vscode-tools.reloadSchemas"
+        "onCommand:azurerm-vscode-tools.reloadSchemas",
```

Comment:
If we keep all these commands, they all need to have an onCommand to wake up the extension.

Reply:
Fixed.